src/Factory/NonceFactory.php

```diff
@@ -39,4 +39,14 @@ protected function _createNonceInstance($id, $code)
     {
         return new Nonce($id, $code);
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @since [*next-version*]
+     */
+    protected function _generateNonceCode($id)
```

Comment:
`AbstractNonceFactory`, the ancestor of this class, already [implements](https://github.com/RebelCode/wp-nonce/blob/develop/src/Factory/AbstractNonceFactory.php#L50) this method.

Reply:
That's on `develop`. See [`AbstractNonceFactory`][1] on this PR's branch.

[1]: https://github.com/RebelCode/wp-nonce/blob/task/improvements/src/Factory/AbstractNonceFactory.php

Reply:
Yep, you're right. And this is correct: WP specifics should be in concrete. <3